docs/authors.rst

```diff
@@ -0,0 +1,24 @@
+=======
+Credits
+=======
+
```

Comment:
It would be nice to say here that the project is maintained by Scrapinghub, and mention the author_mail from setup.py (`info@scrapinghub.com`) as a point of contact.